src/Build/BackEnd/Components/Communications/TranslatorExtensions.cs

```diff
@@ -85,9 +85,9 @@ public static T FactoryForDeserializingTypeWithName<T>(this ITranslator translat
                 {
                     ConstructorInfo constructor = null;
                     constructor = type.GetConstructor(BindingFlags.Instance | BindingFlags.NonPublic, null, Type.EmptyTypes, null);
-                    ErrorUtilities.VerifyThrowInvalidOperation(
+                    ErrorUtilities.VerifyThrow(
                         constructor != null,
-                        $"{typeName} must have a private parameterless constructor");
```

Comment:
Note that interpolated string in this case a is a perf antipattern. It formats the string even if the condition is false. That is exactly why there are VerifyThrow overloads that take args.